src/game/client/components/scoreboard.cpp

```diff
@@ -679,6 +681,47 @@ void CScoreboard::RenderRecordingNotification(float x)
 	TextRender()->Text(0, x+50.0f, 10.0f, 20.0f, aBuf, -1.0f);
 }
 
+void CScoreboard::RenderNetworkQuality(float x, float w)
+{
+	//draw the box
+	CUIRect RectBox = {x, 0.0f, w, 50.0f};
+	vec4 Color = vec4(0.0f, 0.0f, 0.0f, 0.4f);
+	const float LineHeight = 17.0f;
+	int Score = Client()->GetInputtimeMarginStabilityScore();
+
+	Graphics()->BlendNormal();
+	RenderTools()->DrawUIRect(&RectBox, Color, CUI::CORNER_B, 15.0f);
+	Graphics()->TextureSet(g_pData->m_aImages[IMAGE_NETWORKICONS].m_Id);
+	Graphics()->QuadsBegin();
+	RenderTools()->SelectSprite(SPRITE_NETWORK_GOOD);
+	IGraphics::CQuadItem QuadItem(x+20.0f, 12.5f, 25.0f, 25.0f);
+	Graphics()->QuadsDrawTL(&QuadItem, 1);
+	Graphics()->QuadsEnd();
+
+	x += 50.0f;
+	TextRender()->Text(0, x, 10.0f, 20.0f, "NET", -1.0f);
```

Comment:
Localize?

Reply:
That's a tough one. This UI code requires a max 3 characters string, and I'm worried translators will use more characters, and that it would do more harm than good. "REC" is localized though. I see those options

- Localize NET and hope for the best
- Delocalize REC
- Write some code to dynamically force a max 3 characters string.
- Write some code to force max 3 characters in Transifex, or do it manually, or just write a precise description in the second argument of Localize

Reply:
Would vote for filling up the string with ` ` so we render 3 characters there.